back/guide.cabal

```diff
@@ -51,6 +51,7 @@ library
     Guide.App
     Guide.Api
       Guide.Api.Methods
+      Guide.Api.Migration
```

Comment:
This module is not related to the API, so a better name would be appropriate. E.g. `Guide.Db.Schema` (since there are no migrations here it should hardly be called "migration").